instrumentation/opensearch/opensearch-rest-1.0/javaagent/src/test/java/OpenSearchRestTest.java

```diff
@@ -72,6 +73,7 @@ static void setUpOpenSearch()
                 httpClientBuilder ->
                     httpClientBuilder
                         .setSSLContext(sslContext)
+                        .setSSLHostnameVerifier(new NoopHostnameVerifier())
```

Comment:
can you add a code comment explaining why this is needed, to avoid someone removing it later thinking it's not needed (and we don't have CI tests that will fail if it's removed)